bundles/org.openhab.binding.govee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,22 +11,14 @@
 		<channels>
 			<channel id="color" typeId="system.color"/>
 			<channel id="color-temperature" typeId="system.color-temperature"/>
-			<channel id="color-temperature-abs" typeId="color-temperature-abs"/>
+			<channel id="color-temperature-abs" typeId="system.color-temperature-abs"/>
 		</channels>
-		<config-description-ref uri="thing-type:govee:govee-light"/>
-	</thing-type>
 
+		<properties>
+			<property name="thingTypeVersion">1</property>
```

Comment:
What is the thingTypeVersion for?

Reply:
This is part of the OH core automatic mechanism for telling items that had been created on a prior channel type to adapt themselves to a new channel type. It is a monotonically increasing version id to mark different updates of the xml.